frameworks/cassandra/src/main/java/com/mesosphere/sdk/cassandra/scheduler/CassandraRecoveryPlanOverrider.java

```diff
@@ -81,6 +68,15 @@ private Phase getNodeRecoveryPhase(Plan inputPlan, int index) {
         TaskSpec taskSpec = podSpec.getTasks().stream().filter(t -> t.getName().equals("server")).findFirst().get();
         CommandSpec command = taskSpec.getCommand().get();
 
+        // Get IP address for the pre-existing node.
+        Protos.TaskStatus status = StateStoreUtils.getTaskInfoFromProperty(
+                stateStore, TaskSpec.getInstanceName(podInstance, taskSpec)).get();
+
+        String replaceIp = status.getContainerStatus()
+                .getNetworkInfos(0)
```

Comment:
Although unknown right now, it's possible to have >1 `NetworkInfo` on a `ContainerStatus` if the pod is one more than 1 network (e.g. Calico and `dcos`)